controllers/util/k8sutil.go

```diff
@@ -37,7 +37,7 @@ const (
 	EnableTopologyKey           = "CSI_ENABLE_TOPOLOGY"
 	TopologyDomainLabelsKey     = "CSI_TOPOLOGY_DOMAIN_LABELS"
 	EnableNFSKey                = "ROOK_CSI_ENABLE_NFS"
-	CsiRemoveHolderPodsKey      = "CSI_REMOVE_HOLDER_PODS"
+	CsiRemoveHolderPodsKey      = "CSI_DISABLE_HOLDER_PODS"
```

Comment:
I think we should also change the variable name to `CsiDisableHolderPodsKey`.

Reply:
Updated.